app-modules/engagement/src/Notifications/EngagementFailedNotification.php

```diff
@@ -69,10 +70,16 @@ public function toEmail(object $notifiable): MailMessage
 
     public function toDatabase(object $notifiable): array
     {
+        $engagementType = match ($this->engagement->deliverable->channel) {
+            EngagementDeliveryMethod::Email => 'Email',
+            EngagementDeliveryMethod::Sms => 'SMS',
+            default => ''
+        };
+
         return FilamentNotification::make()
             ->danger()
             ->title('Engagement Delivery Failed')
-            ->body("Your engagement failed to be delivered to {$this->engagement->recipient->display_name}.")
+            ->body("Your engagement {$engagementType} failed to be delivered to {$this->engagement->recipient->display_name}.")
```

Comment:
```suggestion
            ->body("Your {$engagementType} engagement failed to be delivered to {$this->engagement->recipient->display_name}.")
```

Reply:
Hmm, it reads a little awkward to me: 

"Your engagement email was..." vs "Your email engagement was..."
"Your engagement sms was..." vs "Your sms engagement was..."

@Orrison What do you think?

Reply:
What we have here now is fine.